mempool/iterators_test.go

```diff
@@ -399,3 +399,46 @@ func TestReapOrderMatchesGossipOrder(t *testing.T) {
 	require.NoError(t, err)
 	require.Zero(t, mp.Size())
 }
+
+func TestReapOrderMatchesGossipOrderNolanes(t *testing.T) {
```

Comment:
Does this test belong to this PR?

Reply:
Ok, I thought the PR was only about e2e.

Reply:
I am creating a new PR for this ,it is cleaner